lib/js/components/Banner/Banner.consts.ts

```diff
@@ -0,0 +1,13 @@
+export const BANNER_COLORS = {
+	DEFAULT: 'default',
+	NEUTRAL: 'neutral',
+	SUCCESS: 'success',
+	FAIL: 'fail',
+	WARNING: 'warning',
+	INFO: 'info',
```

Comment:
In Figma I see only 4 colors: neutral, warning, info and inverted.
In Miro there are 4 colors shown and 6 colors listed in the table, different than in Figma: default (biały), neutral, success, fail, warning, info.

Where do you get your source of truth?